packages/ozone/src/db/schema/repo_push_event.ts

```diff
@@ -0,0 +1,18 @@
+import { Generated } from 'kysely'
+
+export const eventTableName = 'repo_push_event'
+
+export type RepoPushEventType = 'takedown'
+
+export interface RepoPushEvent {
```

Comment:
I opted to use separate tables here so that we can enforce constraints about the data we need (like uri in the case of a record)

I've gone back and forth on where I should capture the fact that a takedown gets fanned out to multiple services. ie:
- fanout on write to push_event table with different event types: `pds_takedown` & `appview_takedown`
- or fanout on read from push_event table

I implemented the former mostly for simplicities sake, but am now leaning towards the latter for better error tracking - any thoughts?

Reply:
It looks like in the current state it fans out on read.  To be honest I lean a little bit towards fanout on write.  I have two reasons for that:
 - pushes represent a single push to a single service.  That strikes me as being simpler and allows them to fail independently, which gives us strictly more information.
 - it's more flexible, will be more natural to adapt if we need to start taking different actions on pds versus appview.  Or maybe more likely, take an action on one but not the other.

Reply:
kk that was my feeling as well - i switched it over 👍